test/definitions/test_helper.rb

```diff
@@ -84,6 +84,9 @@ def refute_scenario(filter)
 end
 
 TEST_DIR = File.dirname(__FILE__)
-ForemanMaintain.setup(
-  :config_file => File.join(TEST_DIR, 'config/foreman_maintain.yml.test')
-)
+CONFIG_FILE = File.join(TEST_DIR, 'config/foreman_maintain.yml.test').freeze
+STORAGE_DATA_FILE = '/lib/foreman-maintain/data_test.yml'.freeze
```

Comment:
I declared constants twice for tests.

I have tried to move these constants into separate file under test folder & that file I have required into `test/definitions/test_helper.rb`  & `test/lib/test_helper.rb`
which results in test case failures. 
Please let me know if any other way around.